data-plane/src/server/layers/decrypt.rs

```diff
@@ -45,19 +52,26 @@ impl std::convert::From<DecryptError> for Response<Body> {
     }
 }
 
+lazy_static! {
+    static ref CIPHERTEXT_VERSION_REGEX: regex::Regex = get_ciphertext_regex();
+}
+
 #[derive(Clone)]
-pub struct DecryptLayer {
-    e3_client: Arc<E3Client>,
+pub struct DecryptLayer<T: E3Api + Send + Sync + 'static>
+where
```

Comment:
I don't think the `where` with trait bounds is also needed here?

Reply:
ahh yes removed!